example/spec/helpers/testUtils.js

```diff
@@ -87,18 +89,23 @@ async function deleteFolder(bucket, folder) {
  */
 function getExecutionUrl(executionArn) {
   const region = process.env.AWS_DEFAULT_REGION || 'us-east-1';
-  return `https://console.aws.amazon.com/states/home?region=${region}` +
-         `#/executions/details/${executionArn}`;
+  return `https://console.aws.amazon.com/states/home?region=${region}`
+         + `#/executions/details/${executionArn}`;
 }
 
 /**
  * Redeploy the current Cumulus deployment
  *
  * @param {Object} config - configuration object from loadConfig()
- * @returns {undefined} none
+ * @param {string} [template=node_modules/@cumulus/deployment/app] - optional template command line kes option
+ * @param {string} [kesClass] - optional kes-class command line kes option
+ * @returns {Promise.undefined} none
  */
-async function redeploy(config) {
-  const deployCommand = `./node_modules/.bin/kes  cf deploy --kes-folder app --template node_modules/@cumulus/deployment/app --deployment ${config.deployment} --region us-east-1`;
+
+async function redeploy(config, template, kesClass) {
+  const templatePath = template || 'node_modules/@cumulus/deployment/app';
+  let deployCommand = `./node_modules/.bin/kes cf deploy --kes-folder app --template ${templatePath} --deployment ${config.deployment} --region us-east-1`;
```

Comment:
I know this isn't for this PR but we probably shouldn't be hard-coding region IMO

Reply:
Except as fallback when not specified

Reply:
Given how many different ways there are to set the region (environment variable, .aws/config, etc), it could be error-prone for us to try to check if region _has_ been set.  I'd say the best bet is to not set it here and expect the person (or script) performing the deployment to make sure that a region has been set.